src/js/components/Accordion.js

```diff
@@ -0,0 +1,84 @@
+// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
+
+import React, { Component, PropTypes } from 'react';
+import classnames from 'classnames';
+import Box from './Box';
+import Headline from './Headline';
+import List from './List';
+import Paragraph from './Paragraph';
+
+import CSSClassnames from '../utils/CSSClassnames';
+
+const CLASS_ROOT = CSSClassnames.ACCORDION;
+
+export default class Accordion extends Component {
```

Comment:
I believe we need Accordion instead of List. We need to add aria-role="tablist" which is specific for components like Tab and Accordion.